Scripts/developer_scripts/cgal_check_dependencies.sh

```diff
@@ -29,6 +31,9 @@ do
 done
 
 cmake -DCGAL_HEADER_ONLY=FALSE -DCGAL_ENABLE_CHECK_HEADERS=TRUE -DDOXYGEN_EXECUTABLE="$DOX_PATH" -DCGAL_COPY_DEPENDENCIES=TRUE -DCMAKE_CXX_FLAGS="-std=c++11" ..
+if [ -n "$DO_CHECK_HEADERS" ]; then
+    make -j$(nprc --all) -k check_headers
```

Comment:
```
Scripts/developer_scripts/cgal_check_dependencies.sh: line 35: nprc: command not found
```
https://travis-ci.org/CGAL/cgal/jobs/385068433#L4956

Reply:
I think that error lead to a run of `make -j`, without any number after the `-j`, and that lead Travis to fork thousands of compilation processes in parallel.

Reply:
there is a 'o' missing :/ I fix that immediatly